pkg/securityscan/jobHandler.go

```diff
@@ -187,8 +195,36 @@ func (c *Controller) createClusterScanReport(outputBytes []byte, scan *v1.Cluste
 
 func (c *Controller) ensureCleanup(scan *v1.ClusterScan) error {
 	var err error
-	configmaps := c.coreFactory.Core().V1().ConfigMap()
+	// Delete the dameonset
+	daemonsets := c.appsFactory.Apps().V1().DaemonSet()
+	dsList, err := daemonsets.Cache().List(v1.ClusterScanNS, labels.NewSelector())
+	if err != nil {
```

Comment:
do we need to check for a particular deployment before deleting like line 218-219 do for pods?

Reply:
We only spawn the security-scan DS in this namespace, but added a prefix check for safe delete